composer.json

```
@@ -56,16 +55,17 @@
     }
   },
   "scripts": {
-    "cs:check": "php-cs-fixer fix -v --diff --dry-run",
-    "cs:fix": "php-cs-fixer fix -v --diff",
-    "phpstan": "phpstan analyse --level=${PHPSTAN_LEVEL:-8} -c phpstan.neon",
-    "phpstan:baseline": "phpstan analyse --level=${PHPSTAN_LEVEL:-8} -c phpstan.neon --generate-baseline",
+    "cs:check": "@cs:fix --dry-run",
+    "cs:fix": "php-cs-fixer fix --ansi --verbose --diff",
+    "phpstan": "phpstan analyse -c phpstan.neon --ansi --no-interaction",
+    "phpstan:baseline": "phpstan analyse -c phpstan.neon --generate-baseline",
```

I don't think we need to manually set the configuration here (we don't do it for `php-cs-fixer` either)

```suggestion
    "phpstan": "phpstan analyse --ansi --no-interaction",
    "phpstan:baseline": "phpstan analyse --generate-baseline",
```